tools/WinObjC.Packaging/WinObjC.Packaging.nuproj

```diff
@@ -17,28 +17,28 @@
   <PropertyGroup Label="Globals">
     <OutputPath>$(SolutionDir)\OutputPackages\$(Configuration)\</OutputPath>
     <ProjectGuid>ddc2aa5f-c8ae-48cb-adb7-9278985c3db7</ProjectGuid>
+    <SkipGetPackageContentsForOtherPlatforms>true</SkipGetPackageContentsForOtherPlatforms>
   </PropertyGroup>
   <PropertyGroup Label="Configuration" />
   <ItemGroup>
-    <None Include="Package.Default.props" />
-    <None Include="Package.Default.targets" />
-    <None Include="Package.props" />
-    <None Include="Package.targets" />
-    <None Include="WinObjC.Packager.props" />
-    <None Include="WinObjC.Packager.targets" />
-    <None Include="WinObjC.Packageable.props" />
-    <None Include="WinObjC.Packageable.targets" />
-    <None Include="project.json" />
-    <None Include="ImportedByThisProj.targets" />
-    <None Include="ImportedByThisProj.props" />
-    <None Include="ImportedByAnyDependent.targets" />
-    <None Include="ImportedByAnyDependent.props" />
-    <None Include="Readme.txt">
+    <PackageFile Include="Package.Default.props" />
```

Comment:
>PackageFile [](start = 5, length = 11)

same.

Reply:
all of these should be None with IncludeInPackage

---
In reply to: [141463658](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141463658) [](ancestors = 141463658)

Reply:
ok actually not all. There are two here that help describe how to package this project (ImportedByThisProj.*) that don't belong in the package since they are only used in how to make this pacakge.

---
In reply to: [141463797](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141463797) [](ancestors = 141463797,141463658)